PowerEditor/src/ScintillaComponent/ScintillaEditView.cpp

```diff
@@ -340,6 +340,8 @@ void ScintillaEditView::init(HINSTANCE hInst, HWND hPere)
 	}
 	//Get the startup document and make a buffer for it so it can be accessed like a file
 	attachDefaultDoc();
+
+	_pGetExceptionMsgFunc = reinterpret_cast<GetExceptionMessageFunc>(::GetProcAddress(_hLib, "GetBoostExceptionMessage"));

```

Comment:
With the [fusion of notepad++.exe & SciLexer.dll](https://github.com/notepad-plus-plus/notepad-plus-plus/pull/9594) (coming soon), there won't be `_hLib` to get process' address. What'll be the solution?